src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -1,19 +1,38 @@
+import { ReactNode } from "react"
 import styles from "./Breadcrumb.module.css"
-import { BreadcrumbItem, LinkProps } from "./BreadcrumbItem"
+import { BreadcrumbItem } from "./BreadcrumbItem"
+
+export interface Link {
+  href: string
+  label: string
+  onClick?: () => void
+}
+
+export interface LinkElement {
+  renderItem?: ({
+    item,
+    isCurrent,
+  }: {
+    item: Link
+    isCurrent: boolean
+  }) => ReactNode
+}
```

Comment:
✋ I think a caller might be thrown off by the nesting of the render prop within an object. Any reason we wouldn't do this:
```suggestion
export type RenderItem = ({
    item,
    isCurrent,
  }: {
    item: Link
    isCurrent: boolean
  }) => ReactNode
```

Reply:
@roblafeve I like this!